src/main/resources/data/spectrum/recipes/titration_barrel/bristle_mead_fluid.json

```diff
@@ -14,7 +14,7 @@
       "count": 2
     }
   ],
-  "fluid": "create:honey",
+  "fluid": "c:honey",
```

Comment:
Pretty sure the Titration Barrel does not support fluid tags right now. That would definitely be a nice addition for mod compat.

Reply:
oh okay, my bad